_includes/nav.html

```diff
@@ -24,7 +24,11 @@
             <a class="navbar-brand" href="{{home}}/">
               <div>
                 <img src="{{home}}/img/logo.png" alt="Istio" width="36px" height="54px"/>
-                <span class="brand-name">Istio</span>
+                {% if site.data.istio.archive %}
+                  <span class="brand-name">Istioldie {{site.data.istio.short_version}}</span>
```

Comment:
`Istioldie` was fine/fun for the top level of https://archive.istio.io/ but can we just have
`Istio archive v{{site.data.istio.short_version}}` here
  
Also I made it smaller than Istio in https://archive.istio.io/v0.2/docs/ for instance, which I think is useful
(add another span/class for the archive tag)